app/image/color_format.py

```diff
@@ -61,20 +61,49 @@ def bits_per_components(self, bpcs):
             "Provided value should be an iterable of 3 or 4 values!")
 
     def __str__(self):
-        return name
-
-
-RGB3_FORMAT = ColorFormat(PixelFormat.RGBA,
-                          Endianness.BIG_ENDIAN,
-                          PixelPlane.PACKED,
-                          8,
-                          8,
-                          8,
-                          name="RGB3")
-BGR3_FORMAT = ColorFormat(PixelFormat.BGRA,
-                          Endianness.BIG_ENDIAN,
-                          PixelPlane.PACKED,
-                          8,
-                          8,
-                          8,
-                          name="BGR3")
+        return self.name
+
+
+AVAILABLE_FORMATS = {
+    'RGB24':
+    ColorFormat(PixelFormat.RGBA,
+                Endianness.BIG_ENDIAN,
+                PixelPlane.PACKED,
+                8,
+                8,
+                8,
+                name="RGB24"),
+    'BGR24':
+    ColorFormat(PixelFormat.BGRA,
+                Endianness.BIG_ENDIAN,
+                PixelPlane.PACKED,
+                8,
+                8,
+                8,
+                name="BGR24"),
+    'RGB32':
+    ColorFormat(PixelFormat.RGBA,
+                Endianness.BIG_ENDIAN,
+                PixelPlane.PACKED,
+                8,
+                8,
+                8,
+                8,
+                name="RGB32"),
+    'RGB332':
+    ColorFormat(PixelFormat.RGBA,
+                Endianness.BIG_ENDIAN,
```

Comment:
This format is little-endian - [src](https://linuxtv.org/downloads/v4l-dvb-apis/userspace-api/v4l/pixfmt-rgb.html#less-than-8-bits-per-component)